### PR TITLE
DASH-131 Implement additional Ledger Actions

### PR DESCRIPTION
**Pull request by vmkarpenko**

### CODE REVIEW

**Review by NikolasStelmakh — APPROVED** [review]

File: client/src/pages/FioAddressAddBundlesPage/components/AddBundlesLedgerWallet.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+  const submit = async (appFio: LedgerFioApp) => {
+    const { chainId, transaction } = await prepareChainTransaction(
+      fioWallet.publicKey,
+      ACTIONS.addBundledTransactions,
+      {
+        fio_address: submitData.fioAddress,
+        bundle_sets: submitData.bundleSets,
+        max_fee: fee,
+        tpid: '',
```
Comment (trukhilio) [inline client/src/pages/FioAddressAddBundlesPage/components/AddBundlesLedgerWallet.tsx:48]: Same tpid here

File: client/src/pages/FioDomainStatusChangePage/components/FioDomainStatusLedgerWallet.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    const { statusToChange, name } = submitData;
+
+    const { chainId, transaction } = await prepareChainTransaction(
+      fioWallet.publicKey,
+      ACTIONS.setFioDomainVisibility,
+      {
+        fio_domain: name,
+        is_public: statusToChange === DOMAIN_STATUS.PUBLIC ? 1 : 0,
+        max_fee: fee,
+        tpid: '',
```
Comment (trukhilio) [inline client/src/pages/FioDomainStatusChangePage/components/FioDomainStatusLedgerWallet.tsx:53]: Same tpid here

File: client/src/pages/FioTokensRequestPage/components/RequestTokensLedgerWallet.tsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+          .toUncompressed()
+          .toBuffer()
+          .toString('hex'),
+        payee_public_address: fioWallet.publicKey,
+        amount: submitData.amount,
+        chain_code: submitData.chainCode,
+        token_code: submitData.tokenCode,
+        memo: submitData.memo,
+        max_fee: fee || 0,
+        tpid: '',
```
Comment (trukhilio) [inline client/src/pages/FioTokensRequestPage/components/RequestTokensLedgerWallet.tsx:71]: Same tpid here

File: client/src/pages/RejectFioRequestPage/components/RejectRequestLedger.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    try {
+      const { chainId, transaction } = await prepareChainTransaction(
+        fioWallet.publicKey,
+        ACTIONS.rejectFundsRequest,
+        {
+          fio_request_id: submitData.fioRecord.id
+            ? submitData.fioRecord.id.toString()
+            : '',
+          max_fee: fee || 0,
+          tpid: '',
```
Comment (trukhilio) [inline client/src/pages/RejectFioRequestPage/components/RejectRequestLedger.tsx:51]: Same tpid here

File: client/src/pages/SendPage/components/SendLedgerWallet.tsx [hunk 5]
```diff
@@ -37,55 +42,123 @@ const SendLedgerWallet: React.FC<Props> = props => {
...
-      sendData.nativeAmount,
-      fee,
+    const { chainId, transaction } = await prepareChainTransaction(
+      fioWallet.publicKey,
+      ACTIONS.transferTokens,
+      {
+        payee_public_key: sendData.toPubKey,
+        amount: new MathOp(sendData.nativeAmount).toNumber(),
+        max_fee: fee,
+        tpid: '',
```
Comment (trukhilio) [inline client/src/pages/SendPage/components/SendLedgerWallet.tsx:59]: Same tpid here

File: client/src/pages/SendPage/components/SendLedgerWallet.tsx [hunk 6]
```diff
@@ -37,55 +42,123 @@ const SendLedgerWallet: React.FC<Props> = props => {
...
+            chain_code: FIO_CHAIN_CODE,
+            token_code: FIO_CHAIN_CODE,
+            obt_id: result.transaction_id,
+            memo: sendData.memo,
+            status: 'sent_to_blockchain',
+            fio_request_id: sendData.fioRequestId
+              ? sendData.fioRequestId.toString()
+              : '',
+            max_fee: feeRecordObtData || 0,
+            tpid: '',
```
Comment (trukhilio) [inline client/src/pages/SendPage/components/SendLedgerWallet.tsx:120]: Same tpid here

File: client/src/pages/TokensRequestPaymentPage/components/PaymentDetailsLedgerWallet.tsx [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+        chain_code: FIO_CHAIN_CODE,
+        token_code: FIO_CHAIN_CODE,
+        obt_id: submitData.obtId,
+        memo: submitData.memo,
+        status: 'sent_to_blockchain',
+        fio_request_id: submitData.fioRequestId
+          ? submitData.fioRequestId.toString()
+          : '',
+        max_fee: fee || 0,
+        tpid: '',
```
Comment (trukhilio) [inline client/src/pages/TokensRequestPaymentPage/components/PaymentDetailsLedgerWallet.tsx:68]: Same tpid here

File: client/src/pages/UnstakeTokensPage/components/UnstakeTokensLedgerWallet.tsx [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+  const unstake = async (appFio: LedgerFioApp) => {
+    const { chainId, transaction } = await prepareChainTransaction(
+      fioWallet.publicKey,
+      ACTIONS.unStakeFioTokens,
+      {
+        amount: apis.fio.amountToSUF(new MathOp(submitData.amount).toNumber()),
+        fio_address: submitData.fioAddress,
+        max_fee: fee,
+        tpid: '',
```
Comment (trukhilio) [inline client/src/pages/UnstakeTokensPage/components/UnstakeTokensLedgerWallet.tsx:48]: Same tpid here

**Review by trukhilio — CHANGES_REQUESTED** [review]
Check comments

**Review by trukhilio — APPROVED** [review]

**Pull request merged**